modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,253 @@
+metadata:
+  name: argo-rollouts-stable-service #<1>
+spec:
+  ports:
+  - port: 80
+    targetPort: http
+    protocol: TCP
+    name: http
+
+  selector: # this field/sub-field must have the same value as Rollout 
+    app: rollouts-demo 
+    # Rollouts will update the Service with pod-template-hash of the stable (non-canary) ReplicaSet. e.g.:
+    # rollouts-pod-template-hash: 1b6a7733
+----
+<1> Name of the stable service created in the Services page
+.. Click *Create* to create the stable service.
++
+[NOTE]
+====
+Ensure that the name of the stable service created in the *Create Route* page matches with the name of the stable service created in the *Create Services* page.
+====
+. Create the `Rollout` CR to reference the Route and the Services.
+.. In the Administrator perspective of the web console, navigate to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps* -> *Rollout*.
+.. On the *Create Rollout* page, click *YAML view* and add the following snippet:
++
+.Example: Creating a `Rollout`
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  template:
+    metadata:
+      labels:
+        app: rollouts-demo
+    spec:
+      containers:
+      - name: rollouts-demo
+        image: argoproj/rollouts-demo:blue
+        ports:
+        - name: http
+          containerPort: 8080
+          protocol: TCP
+        resources:
+          requests:
+            memory: 32Mi
+            cpu: 5m
+                
+  revisionHistoryLimit: 2
+  replicas: 5
+  strategy:
+    canary:
+      canaryService: argo-rollouts-canary-service # must match name of Service above #<1>
+      stableService: argo-rollouts-stable-service # must match name of Service above #<2>
+      trafficRouting:
+        plugins:
+          argoproj-labs/openshift:          
+            routes: #<3>
+              - rollouts-demo-route  # must match name of Route, above
+      steps: # In this example, route 30%/60%/100% of traffic to canary
+      - setWeight: 30
+      - pause: {}
+      - setWeight: 60
+      - pause: {}
+  selector: # Contents of this field must match the same value in the canary and stable Services (same contents for both)
+    matchLabels:
+      app: rollouts-demo
+----
+<1> Rollout references the canary service
+<2> Rollout references the stable service
+<3> Rollout references the route
+.. Click *Create*. 
+.. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the *RolloutManager* instance shows as *Phase: Available*.
++
+When the first instance of the Rollout resource is created, the Rollout regulates the amount of traffic to be directed towards the various versions of the application, i.e. the stable and the canary versions. In the initial instance, the creation of the Rollout resource routes 100% of the traffic towards the stable version of the application and skips the part where the traffic is sent to the canary version.
```

Comment:
```suggestion
When the first instance of the `Rollout ` resource is created, the rollout regulates the amount of traffic to be directed towards the stable and canary application versions. In the initial instance, the creation of the `Rollout` resource routes all of the traffic towards the stable version of the application and skips the part where the traffic is sent to the canary version.
```

Reply:
Woud this information be better to include in the introduction rather than the procedure?

Reply:
Yeah, I agree that this information can also be included in the introduction. Fixed.